<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Abhilash" name="Edit Damage Model" tests="1" failures="1" timestamp="16 Sep 2015 09:38:34 GMT" time="45.883" errors="0">
  <testcase name="EditDamageModel" time="29.349" classname="com.eigenRisk.TestSuite.Damage_ModelManager">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 3, Size: 3">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 3, Size: 3
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at com.eigenRisk.functionalities.DamageModelManager.findDamageModel(DamageModelManager.java:971)
	at com.eigenRisk.functionalities.DamageModelManager.editDamageModel(DamageModelManager.java:200)
	at com.eigenRisk.TestSuite.Damage_ModelManager.EditDamageModel(Damage_ModelManager.java:197)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- EditDamageModel -->
</testsuite> <!-- Edit Damage Model -->
